application/templates/mail/layout.plush.html

```diff
@@ -21,6 +21,11 @@
 			<%= yield %>
```

Comment:
Should there be a bit of a WeCarry intro, in addition to the Logo?  I'm concerned that messages that start with a Post Title could confuse people if that title happens to include funny wording, e.g. "Amazon Delivery".

Reply:
I'll update the subjects of the emails to cover that enough that the body contents can be what they are